daft/dataframe/dataframe.py

```diff
@@ -342,7 +342,7 @@ def iter_partitions(
                 available on the machine).
 
         >>> import daft
-        >>>
+        >>> _ = daft.context.set_runner_py()
```

Comment:
There are doc examples like this one that use `into_partitions`, so it will fail on swordfish because the outputs don't match.

Reply:
> There are doc examples like this one that use `into_partitions`, so it will fail on swordfish because the outputs don't match. Still thinking of a solution